ui/admin/app/routes/scopes/scope/host-catalogs/index.js

```diff
@@ -4,8 +4,88 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
 export default class ScopesScopeHostCatalogsIndexRoute extends Route {
+  // =services
+
+  @service store;
+  @service can;
+
+  // =attributes
+
+  queryParams = {
+    search: {
+      refreshModel: true,
+      replace: true,
+    },
+    page: {
+      refreshModel: true,
+    },
+    pageSize: {
+      refreshModel: true,
+    },
+  };
+
+  // =methods
+
+  /**
+   * Loads all host catalogs under the current scope.
+   * @returns {Promise<{totalItems: number, hostCatalogs: [HostCatalogModel] }> }   */
```

Comment:
non-blocking:
Why is the format here a little different. The `*/` is usually on a new line. `hostCatalogsExist` is missing here here as well.

Reply:
Fixed